config/default.json

```diff
@@ -44,6 +44,9 @@
         "WebserverMode": "gzip",
         "EnableCustomEmoji": false,
         "EnableEmojiPicker": true,
+        "EnableGifPicker": true,
```

Comment:
If the api keys aren't specified and the gif picker is enabled, does this cause any errors or anything in the UI?

Reply:
This has been resolved in the redux repo